dali/pipeline/operators/decoder/nvjpeg_allocator.cc

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/pipeline/operators/decoder/nvjpeg_allocator.h"
```

Comment:
wonder if the name could be a little more general, only in case this is not tightly coupled to nvjpeg

Reply:
For now it is tightly coupled to nvjpeg, not really flexible, its not a malloc implem